api/v1beta1/deployment_types.go

```diff
@@ -52,13 +62,42 @@ type DeploymentStrategy struct {
 	Partition intstr.IntOrString `json:"partition,omitempty"`
 }
 
+// OriginalSetting storages part of the fileds of a workload,
```

Comment:
// DeploymentStrategy stores deployment strategy related fields of a workload

Reply:
updaed